### PR TITLE
`cuda.core` v0.1.1 final doc touch

### PR DESCRIPTION
**Pull request by leofang**

### CODE REVIEW

File: cuda_core/docs/source/release/0.1.1-notes.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# `cuda.core` Release notes
+# `cuda.core` v0.1.1 Release notes
 
 Released on Dec XX, 2024
```
Comment (leofang) [inline cuda_core/docs/source/release/0.1.1-notes.md]:
```suggestion
Released on Dec 20, 2024
```

**Comment by leofang** [comment]
@ksimpson-work @jollylili let's merge this and rebuild the docs

**Review by jollylili — APPROVED** [review]

File: cuda_core/docs/source/release/0.1.1-notes.md [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
 
-Released on Dec XX, 2024
+Released on Dec 20, 2024
 
 ## Hightlights
 
 - Add `StridedMemoryView` and `@args_viewable_as_strided_memory` that provide a concrete
   implementation of DLPack & CUDA Array Interface supports.
 - Add `Linker` that can link one or multiple `ObjectCode` instances generated by `Program`s. Under
-  the hood, it uses either the nvJitLink or cuLink APIs depending on the CUDA version detected
```
Comment (keenan-simpson) [inline cuda_core/docs/source/release/0.1.1-notes.md:10]: I think the we can leave 'Program' singular, but I don't have a strong preference

File: cuda_core/docs/source/release/0.1.1-notes.md [hunk 3]
```diff
@@ -1,25 +1,28 @@
...
-  in the current environment.
-- Add a `cuda.core.experimental.system` module for querying system- or process- wide information.
-- Support TCC devices with a default synchronous memory resource to avoid the use of memory pools
+  the hood, it uses either the nvJitLink or driver (`cuLink*`) APIs depending on the CUDA version
+  detected in the current environment.
+- Support `pip install cuda-core`. Please see the Installation Guide for further detail.
```
Comment (keenan-simpson) [inline cuda_core/docs/source/release/0.1.1-notes.md]: further details

**Review by ksimpson-work — APPROVED** [review]
LGTM, couple nits but feel free to ignore if you disagree

File: cuda_core/docs/source/release/0.1.1-notes.md (hunk 2 above)
Comment (leofang) [reply cuda_core/docs/source/release/0.1.1-notes.md:10]: done in commit [3641e27](https://github.com/NVIDIA/cuda-python/pull/301/commits/3641e2730efcb0fc63c57e1a406c57e207fde102)

File: cuda_core/docs/source/release/0.1.1-notes.md (hunk 3 above)
Comment (leofang) [reply cuda_core/docs/source/release/0.1.1-notes.md]: done in commit [3641e27](https://github.com/NVIDIA/cuda-python/pull/301/commits/3641e2730efcb0fc63c57e1a406c57e207fde102)

**Review by ksimpson-work — APPROVED** [review]

**Pull request merged**